virttest/qemu_devices/qdevices.py

```diff
@@ -2430,3 +2514,82 @@ def _update_device_props(self, device, addr):
     def _set_device_props(self, device, addr):
         """ Change value to drive{A,B,...} """
         device.set_param('index', self._addr2stor(addr))
+
+
+class QCPUBus(QSparseBus):
+    """
+    CPU bus representation including cpu/cpu-core handling.
+    """
+
+    def __init__(self, cpu_driver, vcpus_count, addr_spec, aobject=None,
+                 atype=None):
+        """
```

Comment:
What's addr_spec?

Reply:
If you think it's not clear, I can split it apart with cpu_props and props_lengths

Reply:
You can refer to the base class for details.